meson_options.txt

```diff
@@ -0,0 +1,16 @@
+option('MFX_DECODER', type : 'boolean', value : true)
+option('USE_HEVC_DECODER', type : 'boolean', value : true)
+option('USE_VP8_DECODER', type : 'boolean', value : true)
+option('USE_VP9_DECODER', type : 'boolean', value : true)
```

Comment:
Default  to 'false' to prevent  potential compilation issues with older systems.